### ISSUE TITLE
Update this extension for compatibility with newer versions of GNOME Shell

### ISSUE THREAD

**Issue by alemairebe** [getting-things-gnome/gtg-gnome-shell-extension#11]
Hi! Are there any plans to port the GTG extension to GS 3.8?

Thanks for the work done.

**Comment by bsaleil** [comment]
Yes, but I don't really have free time yet. Contributions are also welcome.

**Comment by bsaleil** [comment]
We need to try the extension with 3.8, check what are the API changes from 3.6 with 'LookingGlass' then modify extension to match the new gnome-shell API.

**Comment by nekohayo** [comment]
I don't know if @bsaleil has any free time nowadays (?), or if someone here would like to adopt this extension and update it for the upcoming GNOME 40? GTG itself has been revived and is actively maintained now, at least there is a guarantee that your contributions to the extension are not going to be stuck in the void :)
